curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md

```diff
@@ -21,12 +21,12 @@ By considering the terms in the Fibonacci sequence whose values do not exceed <c
 
 ```yml
 tests:
-  - text: <code>fiboEvenSum(10)</code> should return 188.
-    testString: assert.strictEqual(fiboEvenSum(10), 188, '<code>fiboEvenSum(10)</code> should return 188.');
+  - text: <code>fiboEvenSum(10)</code> should return 44.
+    testString: assert.strictEqual(fiboEvenSum(10), 44, '<code>fiboEvenSum(10)</code> should return 44.');
   - text: <code>fiboEvenSum(23)</code> should return 60696.
     testString: assert.strictEqual(fiboEvenSum(23), 60696, '<code>fiboEvenSum(23)</code> should return 60696.');
-  - text: <code>fiboEvenSum(43)</code> should return 1485607536.
-    testString: assert.strictEqual(fiboEvenSum(43), 1485607536, '<code>fiboEvenSum(43)</code> should return 1485607536.');
+  - text: <code>fiboEvenSum(43)</code> should return 350704366.
+    testString: assert.strictEqual(fiboEvenSum(43), 350704366, '<code>fiboEvenSum(43)</code> should return 350704366.');
   - text: Your function is not returning the correct result using our tests values.
```

Comment:
Is there a reason why we're hiding the tests?
Previously discussed, hidden tests was something we we're steering from as its hard for the person to debug. Unless, I missed a discussion where we're allowing hidden tests.

Reply:
Yes, this is a very good point. I'm sure a lot of the other Project Euler problems have this problem, too. I also see that the last test checking for an even number is passing a pretty high number to the function. Would probably be better if we change that.

@reolander, could you change the tests section to the following?

```
tests:
  - text: <code>fiboEvenSum(10)</code> should return 44.
    testString: assert.strictEqual(fiboEvenSum(10), 44, '<code>fiboEvenSum(10)</code> should return 44.');
  - text: <code>fiboEvenSum(18)</code> should return 3382.
    testString: assert.strictEqual(fiboEvenSum(18), 3382, '<code>fiboEvenSum(18)</code> should return 3382.');
  - text: <code>fiboEvenSum(23)</code> should return 60696.
    testString: assert.strictEqual(fiboEvenSum(23), 60696, '<code>fiboEvenSum(23)</code> should return 60696.');
  - text: <code>fiboEvenSum(43)</code> should return 350704366.
    testString: assert.strictEqual(fiboEvenSum(43), 350704366, '<code>fiboEvenSum(43)</code> should return 350704366.');
  - text: Your function should return an <code>even</code> value.
    testString: assert.equal(fiboEvenSum(10) % 2 === 0, true, 'Your function should return an <code>even</code> value.');
```

Reply:
@scissorsneedfoodtoo let me know if there is anything else to be done!